docs/ops/generation/RandomUniform_8.md

```diff
@@ -0,0 +1,233 @@
+## RandomUniform <a name="RandomUniform"></a> {#openvino_docs_ops_generation_RandomUniform_8}
+
+**Versioned name**: *RandomUniform-8*
+
+**Category**: Generation
+
+**Short description**: *RandomUniform* operation generates a sequence of random values from a uniform distribution.
+
+**Detailed description**:
+
+*RandomUniform* operation generates random numbers from a uniform distribution in the range `[*minval*, *maxval*)`. 
+The generation algorithm is based on underlying random integer generator that uses Philox algorithm. Philox algorithm 
+is a counter-based pseudo-random generator, which produces uint32 values. Single invocation of Philox algorithm returns 
+four result random values, depending on the given *key* and *counter* values. *Key* and *counter* are initialized 
+with *seed* and *seed2* attributes respectively.
+
+\f[
+key = seed\\
+counter = seed2
+\f]
+
+Link to the original paper [Parallel Random Numbers: As Easy as 1, 2, 3](https://www.thesalmons.org/john/random123/papers/random123sc11.pdf)
+
+(https://www.thesalmons.org/john/random123/papers/random123sc11.pdf)
```

Comment:
~(https://www.thesalmons.org/john/random123/papers/random123sc11.pdf)~
Now may be deleted as it is  implemented above)

Reply:
Done.